frontend-shared/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@hypothesis/frontend-shared",
+  "version": "1.0.0",
+  "repository": "hypothesis/client/frontend-shared",
+  "devDependencies": {
+    "@babel/cli": "^7.1.6",
+    "@babel/core": "^7.1.6",
+    "@babel/preset-env": "^7.1.6",
+    "@babel/preset-react": "^7.0.0"
+  },
+  "scripts": {
+    "build": "npx babel src --out-dir lib --source-maps --ignore '**/test'"
+  },
+  "files": [
+    "lib",
+    "styles",
+    "images"
+  ],
+  "main": "./lib/index.js",
+  "browserify": {
```

Comment:
This is a bit of the odd coupling between this package and the client. This is exclusively used by the client so it can bundle svg files. Browserify does not exist in the frontend-shared package. I suspect that we need the config here because browserify does not stringify svg files inside this package otherwise. I suspect it detects the existence of a new sub-package (package.json) and basically stops applying transformations. Hence, I believe the config needs to be duplicated.